traffic_ops/traffic_ops_golang/trafficvault/backends/riaksvc/dsutil.go

```diff
@@ -23,6 +23,7 @@ import (
 	"database/sql"
 	"encoding/json"
 	"errors"
+	"github.com/apache/trafficcontrol/lib/go-rfc"
```

Comment:
This should be grouped below with the other `github.com/apache/trafficcontrol/` import